### DISCUSSION TITLE
Bot activation

### DISCUSSION THREAD

**Discussion by Shubham-Gajare01** [jacktheboss220/WhatsAppBotMultiDevice#10] [Q&A] [answered]
How can I add bot to my group and is it free ?

**Comment by jacktheboss220** [comment, marked as answer]
> How can I add bot to my group and is it free ?

yes u can add, yes it is free

**Reply by Shubham-Gajare01** [reply]
Bot ain't working saying "```By default, bot is turned off in this group.
Ask the Owner to activate.

Use ```-dev" i and the owner and creater of that group but still I put any command it only saying this

**Reply by jacktheboss220** [reply]
drop a message on telegram
